### DISCUSSION TITLE
TSA does not process anything

### DISCUSSION THREAD

**Discussion by jo-spek** [davidfrantz/force#211] [Q&A] [answered]
Hi everybody, I am at my wits end. I have processed a large stack of images which are now ARD. Running them through force-higher-level for a time series interpolation does however not work. I have triple- and quadruple-checked the tsi.prm-file and I can't find anything wrong. Also the underlying datacube-definition has been successfully created during ARD processing. The time series analysis does start, but immediately finishes with no products.
![tsi_attempt](https://user-images.githubusercontent.com/93268515/187252613-b81483c4-3889-4375-b421-df632abd8391.png)

Find attached the tsi.prm-file and the datacube-definition.prj.
[tsi_datacube.zip](https://github.com/davidfrantz/force/files/9446431/tsi_datacube.zip)

Hope somebody can help.

P.S.: As I'm already asking here, I want to do an interpolation with bi-weekly steps. In the tutorial it is done with 16 day steps. Would I just change to INT_DAY = 14 and RBF_SIGMA = 7 14 28 ? Would that work out?

**Comment by davidfrantz** [comment, marked as answer]
Hi @jonas269,

this is likely related to one of the filtering options.

This is usually caused by non-matching entries for 

```
DIR_MASK
BASE_MASK
X_TILE_RANGE
Y_TILE_RANGE
FILE_TILE
SENSORS
DATE_RANGE
DOY_RANGE
```

In most cases, this is related to the masks. Can you please verify all these entries?

I answered your PS question in another thread: https://github.com/davidfrantz/force/discussions/212

Cheers,
David

**Reply by jo-spek** [reply]
Hi David,
I did not properly pre-process my mask with force-cube, which caused the problem.
Thanks a lot for your immediate support!!

Cheers, Jonas
